notifier_types.go

```diff
@@ -57,7 +38,7 @@ type DetailedNotifier interface {
 	GetType() NotifierType
 	GetSendAllEvents() bool
 	GetSendAllAlerts() bool
-	GetSelectedEvents() []EventTypeStruct
```

Comment:
As `selected_events` is only deprecated, shouldn't we make this lib compatible with both params: `selected_events` and `selected_event_ids`?

Reply:
The thing is that this params selected_events as []EventTypeStruct has never existed at all, the go library ALWAYS had an `[]` in it, because the API never embeeded the types in the notifier, that's why I allowed myself to change this